movie-subdomain/src/main/kotlin/com/example/ajaxonboardingproject/domain/Movie.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.ajaxonboardingproject.domain
+
+data class Movie(
+    val title: String,
+    val description: String,
+) {
+    lateinit var id: String
```

Comment:
here you can make id nullable and move to constructor

Reply:
Done! ✅